tox.ini

```diff
@@ -1,10 +1,23 @@
 [tox]
-envlist = py26, py27, py32, py33, py34, pypy
+envlist = py26, py27, py32, py33, py34, pypy, gae
 
 [testenv]
+passenv = PYTHONPATH TRAVIS*
 deps= -r{toxinidir}/dev-requirements.txt
 commands=
-   nosetests \
-   []
+    nosetests []
 setenv =
     PYTHONWARNINGS=always::DeprecationWarning
+
+[testenv:gae]
+basepython = python2.7
+deps= 
+    -r{toxinidir}/dev-requirements.txt
+    nosegae
+commands=
+    nosetests \
+        --cover-min-percentage=0 \
+        --with-gae \
+        --gae-application=test/appengine/app.yaml \
+        test/appengine \
+        []
```

Comment:
setup.cfg is currently the config file for nose, what do you recommend for this?